install/helm/agones/values.yaml

```diff
@@ -46,6 +46,8 @@ agones:
   createPriorityClass: true
   priorityClassName: agones-system
   cloudProduct: "auto"
+  system:
+    requireDedicatedNode: false
```

Comment:
have you tried `true` on Autopilot?

Reply:
Nice catch. It won't work for AP. Add the clarification in helm.md